rules/prefer-t-regex.js

```diff
@@ -80,18 +84,27 @@ const create = context => {
 	2. `RegExp` class can't be looked up so the function just checks for the name `RegExp`.
 	*/
 	const isRegExp = lookup => {
-		if (lookup.regex) {
```

Comment:
Could use a guard for `lookup` above here.

Reply:
already added 😄